DashAI/back/app.py

```diff
@@ -1,37 +1,77 @@
 """FastAPI Application module."""
 import logging
 import pathlib
+from typing import Any, Dict, Union
 
 from fastapi import FastAPI
 from fastapi.middleware.cors import CORSMiddleware
 
 from DashAI.back.api.api_v0.api import api_router_v0
 from DashAI.back.api.api_v1.api import api_router_v1
 from DashAI.back.api.front_api import router as app_router
+from DashAI.back.config import DefaultSettings
 from DashAI.back.containers import Container
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 
 
-def _create_path(new_path: str) -> None:
-    full_path = pathlib.Path(new_path)
-    if not full_path.is_absolute():
-        full_path = full_path.expanduser()
+def _create_path_if_not_exists(new_path: str) -> None:
+    """Create a new path if it does not exist."""
+    if not new_path.exists():
+        logging.info("Creating new path: %s.", str(new_path))
+        new_path.mkdir(parents=True)
```

Comment:
Why we need to wrap this method?
I feel like 
```suggestion
    new_path.mkdir(parents=True, exists_ok=True)
```
is the same